src/python/pants/core/util_rules/archive.py

```diff
@@ -170,10 +170,10 @@ async def maybe_extract_archive(
     digest: Digest, tar_binary: TarBinary, unzip_binary: UnzipBinary
 ) -> ExtractedArchive:
     """If digest contains a single archive file, extract it, otherwise return the input digest."""
-    output_dir = "__output"
+    extract_archive_dir = "__extract_archive_dir"
```

Comment:
This was not gratuitous, but needed to avoid folks having to blow away their lmdb stores: for the same Process input this PR changes the output digest. I had to blow away my lmdb store to get tests passing but others should not have to take that drastic measure.